crypto/CMakeLists.txt

```diff
@@ -146,10 +140,17 @@ if (CONFIG_CC3XX_BACKEND)
                              "${NRF_CC3XX_BASE}/include"
                              "${NRF_CC3XX_BASE}/include/mbedtls")
   target_link_libraries(mbedcrypto_cc3xx_imported INTERFACE platform_cc3xx)
-  if (NOT CONFIG_NRF_SECURITY_MULTI_BACKEND)
-    # Only CC3XX backend is selected for mbedtls API.
-    # Thus export the alt headers through Zephyr interface library.
-    zephyr_include_directories(${NRF_CC3XX_BASE}/include/mbedtls
-                               ${NRF_CC3XX_BASE}/include)
-  endif()
+
+  if (NOT CONFIG_NORDIC_SECURITY_BACKEND)
+    target_link_libraries(platform_cc3xx INTERFACE mbedcrypto_cc3xx_imported)
+
+    if (NOT CONFIG_NRF_SECURITY_MULTI_BACKEND)
```

Comment:
This line: https://github.com/nrfconnect/sdk-nrfxlib/blob/4c3b97997ebf041c3936300a280a4061b9f268c4/nrf_security/Kconfig#L345

is inside:
```
if NORDIC_SECURITY_BACKEND
...
 if NRF_SECURITY_MULTI_BACKEND 
...
endif # NORDIC_SECURITY_BACKEND

```

so `CONFIG_NRF_SECURITY_MULTI_BACKEND` is always `n` when `CONFIG_NORDIC_SECURITY_BACKEND` is `n`, so why is this test needed ?

Reply:
You are correct, I will move the if not  if NRF_SECURITY_MULTI_BACKEND outside of the previous if.